Source/Core/DolphinWX/VideoConfigDiag.cpp

```diff
@@ -875,10 +871,6 @@ VideoConfigDiag::VideoConfigDiag(wxWindow* parent, const std::string& title)
                                       wxGetTranslation(dump_efb_desc), vconfig.bDumpEFBTarget));
       szr_utility->Add(CreateCheckBox(page_advanced, _("Free Look"),
                                       wxGetTranslation(free_look_desc), vconfig.bFreeLook));
-#if defined(HAVE_LIBAV) || defined(_WIN32)
-      szr_utility->Add(CreateCheckBox(page_advanced, _("Frame Dumps Use FFV1"),
```

Comment:
If we're keeping the FFV1 option anyways, how about we just default to the old behavior for the pix format? That is, use BGRA if the codec is FFV1, otherwise use YUV. We could have separate code to detect an incompatible format and choose a compatible one.